tokio/src/sync/watch.rs

```diff
@@ -604,8 +604,14 @@ impl<T> Drop for Receiver<T> {
 impl<T> Sender<T> {
     /// Sends a new value via the channel, notifying all receivers.
     ///
-    /// This method fails if the channel has been closed, which happens when
-    /// every receiver has been dropped.
+    /// This method fails if the channel is closed, which is the case when
```

Comment:
Maybe it would be worth adding something like "unlike the other methods for sending" to emphasize that only this method fails in this way.

Reply:
Thank you for pointing that out; I added another note in my amend da8b888, mentioning that the other send methods can be used to send a value even if there isn't any receiver.